content/en/Features/networking/high_availability.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "High availability"
+linkTitle: "High availability"
+date: 2022-03-11
+weight: 2
+draft: false
+description: >
+  High availability to provide resiliency in case of zone outages
+---
+
+## Kubernetes Ingress - high availability
```

Comment:
There is also linkerd high availability field

Reply:
oh gotcha, I missed that one, I updated this section and the diagrams to show this.